packages/utils/src/bigInt/index.ts

```
@@ -0,0 +1,415 @@
+    negative,
+    negativeStr,
+    trimStr,
+    integerStr,
+    decimalStr,
+    fullStr: ''.concat(negativeStr).concat(trimStr)
+  }
+}
+
+export function isE(number) {
+  let str = String(number)
+  return !isNaN(Number(str)) && ~str.indexOf('e')
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)